README.dev.md

```diff
@@ -2,32 +2,61 @@
 Installing the Zulip Development environment
 ============================================
 
-You will need a machine with at least 2GB of RAM available (see
+Currently, you will need a machine with at least 2GB of available RAM (see
```

Comment:
Actually we should probably retest this -- it might be 1GB now that I merged the multithreaded worker branch.